keyboards/mt/ncr80/hotswap/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,47 @@
+    KC_LCTL, KC_LALT,                   KC_SPC,                                      KC_RALT,          KC_RCTL,                      KC_LEFT,  KC_DOWN, KC_RGHT),
+  [1] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RESET,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                _______,
+    _______, _______,                   _______,                                     _______,          _______,                       _______, _______, _______),
+  [2] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                _______,
+    _______, _______,                   _______,                                     _______,          _______,                       _______, _______, _______),
+  [3] = LAYOUT(
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,     _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                                _______,
+    _______, _______,                   _______,                                     _______,          _______,                       _______, _______, _______),
```

Comment:
```suggestion
    _______, _______,                   _______,                                     _______,          _______,                       _______, _______, _______)
```
You don't need all these extra blank layers; they'll automatically be created. There are also too many of them (you only have `#define DYNAMIC_KEYMAP_LAYER_COUNT 3`...)

Reply:
I reused files from another variant of the NCR-80,  hence the extra "baggage". I will change it as suggested.